src/platform/memory.h

```diff
@@ -68,3 +68,106 @@ static inline bool isZero(const void* ptr, unsigned long long size)
     }
     return true;
 }
+
+

```

Comment:
Please move this block up above line 58, because otherwise the tests fail to compile (which define `NO_UEFI` but do not include "uefi.h". It also makes sense to only define them in UEFI build, because they will only work there.

Reply:
This was not changed by me(isZero()), how was the build successful before ?